cli/src/action/keygen.rs

```diff
@@ -148,7 +149,7 @@ fn write_keys(
     public_key_path: PathBuf,
     force_create: bool,
     skip_create: bool,
-    change_permissions: bool,
+    group: Option<GroupOptions>,
```

Comment:
As this previously was always passing true, should this still be an option?  If no option is provided on the command line, the behaviour of this command will change. Given that `Auto` is the previous behaviour, perhaps this should not be optional and the arg should be or-elsed to that.

Reply:
After some discussion, this bahaviour change is intentional.